src/pages/HelloWorld.vue

```diff
@@ -1,18 +1,24 @@
 <template>
   <div class="hello">
     <h1>{{ msg }}</h1>
-    <h2>Essential Links</h2>
+    <h2>Arbitrary, Nonessential Links</h2>
     <ul>
       <li><a href="https://vuejs.org" target="_blank">Core Docs</a></li>
-      <li><a href="https://forum.vuejs.org" target="_blank">Forum</a></li>
-      <li><a href="https://chat.vuejs.org" target="_blank">Community Chat</a></li>
-      <li><a href="https://twitter.com/vuejs" target="_blank">Twitter</a></li>
+      <li><a href="https://forum.vuejs.org" target="_blank">People Who Know What They're Talking About, Talking About It</a></li>
+      <li><a href="https://chat.vuejs.org" target="_blank">People Who Don't Know What They're Talking About, but Still Talking About It</a></li>
+      <li><a href="https://twitter.com/vuejs" target="_blank">The President's Personal BullS**t Horn</a></li>
```

Comment:
Don't submit text with swearing (pls)

Reply:
this change doesn't actually effect the website, but I will still fix it